_posts/2015-10-31-joon.markdown

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Joon
+date: 2015-10-31 19:50:00
+homepage: https://github.com/vormwald/joon
+download: https://github.com/vormwald/joon/archive/master.zip
+demo: http://vormwald.github.io/joon/
+author: Mike Vormwald
+thumbnail: joon.png
+license: MIT
```

Comment:
Hello! Nice theme, please change the license value to `MIT License` to match the formatting for all other themes. Thanks!